docs/quickstart_guide.md

```diff
@@ -0,0 +1,70 @@
+## Quick start guide
+First, set up a **single node** with `stock-only` configuration
+as described in
+[vHive Quickstart](https://github.com/vhive-serverless/vHive/blob/main/docs/quickstart_guide.md).
+Do not start vHive, as it is not needed.
+The relevant commands for a single-node cluster are reproduced here.
```

Comment:
I find this flow to be a bit difficult. Suggesting the reader to start a vhive cluster and then including those commands in to `setup_infrastructure.sh`. I used the vhive scripts to start the cluster and then realised that it is already been done in infra script. Since using `setup_infrastructure.sh` is the recommended way, I think that should be the default way. You can then have a note suggesting the alternate option if one wants to do it on their own.